components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -779,6 +781,7 @@ def main(argv):
         # Validate and load necessary credentials
         if component_name in [
```

Comment:
* Since `component_name` no longer contains *only* component names, perhaps we should rename it to something like `target`?
* Since we're modifying all of these `component_name in [` lines, let's change `[]` into `()` since tuples are more efficient in Python.